lib/spack/spack/cmd/ci.py

```diff
@@ -414,6 +442,9 @@ def ci_rebuild(args):
 
     install_args.extend([
         'install',
+        '--only', 'package',    # ONLY install the job spec
+        '--use-cache',          # Ensure using cache for (dependency) packages
```

Comment:
This is the default and thus isn't required, though maybe it makes sense to be explicit.

Reply:
I *like* being explicit to make things clear.  I can remove it if that's an issue.